app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt

```diff
@@ -165,6 +193,14 @@ class SessionsFragment : Fragment(), Injectable, Findable, OnReselectedListener
         }
     }
 
+    private fun reopenPreviousOpenedItem() {
+        val previousItem = Prefs.previousSessionTabId
+        if (previousItem < 0) return
+
+        binding.sessionsViewPager.currentItem = previousItem
+        initPreviousSessionPrefs()
```

Comment:
Is this need? 🤔

Reply:
If don't init here, move to saved position when user change SessionTabMode.
So I add the init method here.

Reply:
🙆